test/apparmor.bats

```diff
@@ -27,10 +27,10 @@ load_default_apparmor_profile_and_run_a_container_with_it() {
 	setup_test
 	start_crio
 
-	sed -e 's/%VALUE%/runtime\/default/g' "$TESTDATA"/sandbox_config_apparmor.json > "$TESTDIR"/apparmor1.json
-
-	pod_id=$(crictl runp "$TESTDIR"/apparmor1.json)
-	ctr_id=$(crictl create "$pod_id" "$TESTDATA"/container_redis.json "$TESTDIR"/apparmor1.json)
+	jq '.linux.security_context.apparmor_profile = "runtime default"' \
+		"$TESTDATA"/sandbox_config.json > "$TESTDIR"/apparmor1.json
+	pod_id=$(crictl runp "$TESTDATA"/apparmor1.json)
+	ctr_id=$(crictl create "$pod_id" "$TESTDATA"/container_redis.json "$TESTDIR"/sandbox_config.json)
```

Comment:
Not sure how important it is, but you're supposed to use the same sandbox config when creating a container.

Reply:
waht do you mean by this?

Reply:
Here

> pod_id=$(crictl runp "$TESTDATA"/apparmor1.json)

you are using `"$TESTDATA"/apparmor1.json` to create a POD. The same pod config should be used when creating a container (i.e. the last argument of crictl create), but instead you have

> ctr_id=$(crictl create "$pod_id" "$TESTDATA"/container_redis.json "$TESTDIR"/sandbox_config.json)

(which probably doesn't exist).

IOW
```suggestion
	ctr_id=$(crictl create "$pod_id" "$TESTDATA"/container_redis.json "$TESTDIR"/apparmor1.json)
```

Also, I have just noticed that `pod_id=$(crictl runp "$TESTDATA"/apparmor1.json)` should actually be `pod_id=$(crictl runp "$TESTDIR"/apparmor1.json)

Reply:
this is still not fixed AFAICS

Reply:
ok, should be fixed!